purchase_sale_inter_company/models/purchase_order.py

```diff
@@ -3,13 +3,30 @@
 # Copyright 2018-2019 Tecnativa - Carlos Dauden
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import _, models
+from odoo import _, api, fields, models
 from odoo.exceptions import UserError
 
 
 class PurchaseOrder(models.Model):
     _inherit = "purchase.order"
 
+    auto_sale_order_id = fields.Many2one(
```

Comment:
Let's name properly new fields (and the same for the other):

```suggestion
    intercompany_sale_order_id = fields.Many2one(
```

Reply:
done